doc/api/http.md

```diff
@@ -1659,7 +1659,9 @@ const setCookie = response.getHeader('set-cookie');
 ### `response.getHeaderNames()`
 
 <!-- YAML
-added: v7.7.0
+added:
+  - v8.0.0
+  - v7.7.0
```

Comment:
So this feature landed on v7.7.0? Why do we want to add v8.0.0 on this list? I don't think that's an interesting addition, all features of v7.7.0 are on v8.0.0 (and every other releases really).
```suggestion
added: v7.7.0
```

Reply:
No, the methods were added to v8.0.0 and backported to v7.7.0. See PR description and commit message.

Reply:
We do the same for other backports.

Reply:
Do you mean that since there are no "holes" it does not make sense to list v8.0.0. In that case I see your point but the info is a bit misleading.

Reply:
There are other occurrences like this that we might want to fix as well if you have a strong opinion about this:

- https://nodejs.org/api/cli.html#-c---check
- https://nodejs.org/api/url.html#class-url

Personally, I think this is more correct.

Reply:
Node.js 8.0.0 was released on 2017-05-30, Node.js 7.7.0 was released 3 months before that on 2017-02-28. I don't see how it's relevant to our user if the feature landed on the Current release line because of a backport or as a regular cherry-picking, the backport doesn't make it special IMO.


> * https://nodejs.org/api/url.html#class-url

That's different, Node.js v6.13.0 was a LTS release, so it makes sense to list it there as it first landed on v7.0.0.

> * https://nodejs.org/api/cli.html#-c---check

That's an interesting one, v4.2.0 is listed as an LTS release (with the `--check` change)

https://github.com/nodejs/node/blob/86099a375af95f519ae699b68ce101aa20236c5f/doc/changelogs/CHANGELOG_V4.md#L3069-L3077

So one could think that v5.0.0 was the Current release that landed first, but it looks like it was released after v4.2.0, and `--check` change is not listed on its changelog:
https://github.com/nodejs/node/blob/86099a375af95f519ae699b68ce101aa20236c5f/doc/changelogs/CHANGELOG_V5.md#L1591

So yes, I think we should fix that.